pkg/converter/types.go

```diff
@@ -107,6 +107,10 @@ type MergeOption struct {
 	OCI bool
 	// OCIRef enables converting OCI tar(.gz) blob to nydus referenced blob.
 	OCIRef bool
+	// WithReferrer associates a reference to the original OCI manifest.
+	// See the `subject` field description in
+	// https://github.com/opencontainers/image-spec/blob/main/manifest.md#image-manifest-property-descriptions
+	WithReferrer bool
```

Comment:
Could help introduce why nydus image has to refer to the original OCI image?

Reply:
Updated the comment, PTAL.

Reply:
Thanks for the update, looks reasonable.